novawallet/Modules/Transfer/TransferSetup/FeeAssetSelectionPresentable.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import SoraUI
+import SoraFoundation
+
+protocol FeeAssetSelectionPresentable {
+    func showFeeAssetSelection(
+        from view: ControllerBackedProtocol?,
+        utilityAsset: ChainAsset,
+        sendingAsset: ChainAsset,
+        currentFeeAsset: ChainAsset?,
+        onFeeAssetSelect: ((ChainAsset) -> Void)?
+    )
+}
+
+extension FeeAssetSelectionPresentable {
+    func showFeeAssetSelection(
+        from view: ControllerBackedProtocol?,
+        utilityAsset: ChainAsset,
+        sendingAsset: ChainAsset,
+        currentFeeAsset: ChainAsset?,
+        onFeeAssetSelect: ((ChainAsset) -> Void)?
+    ) {
+        let viewModel = createViewModel(
+            utilityAsset: utilityAsset,
+            sendingAsset: sendingAsset,
+            currentFeeAsset: currentFeeAsset,
+            onFeeAssetSelect: onFeeAssetSelect
+        )
+
+        let bottomSheet = SwapNetworkFeeSheetViewFactory.createView(from: viewModel)
+
+        let factory = ModalSheetPresentationFactory(configuration: ModalSheetPresentationConfiguration.nova)
+
+        bottomSheet.controller.modalTransitioningFactory = factory
+        bottomSheet.controller.modalPresentationStyle = .custom
+
+        view?.controller.present(bottomSheet.controller, animated: true)
+    }
+
+    private func createViewModel(
+        utilityAsset: ChainAsset,
+        sendingAsset: ChainAsset,
+        currentFeeAsset: ChainAsset?,
+        onFeeAssetSelect: ((ChainAsset) -> Void)?
+    ) -> SwapNetworkFeeSheetViewModel {
```

Comment:
Look like this bottom sheet became more general. Let's rename to something like EditNetworkFeeSheet